shell/assets/translations/en-us.yaml

```diff
@@ -5482,6 +5482,7 @@ tableHeaders:
   username: Local Username
   value: Value
   version: Version
+  virtualSize: Virtual Size
```

Comment:
Please, move it  in `pkg/harvester/l10n/en-us.yaml`. I left a comment on how to change the labelKey as well.